src/main/webapp/app/modules/login/LoginTopText.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+const LoginTopText = props => (
+    <Typography variant="h6" className="dx26-font-bold" style={{ position: 'absolute', right: "3%", top: "4%" }}>
+        DX26
```

Comment:
I think you can use constant here

Reply:
yes i know...but X from DX26 will have different color as i asked in the chat group hence it is added like that..